platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ConfigurationMigrationManager.java

```diff
@@ -135,9 +137,17 @@ BundleContext getBundleContext() {
         List<MigrationWarning> warnings = new LinkedList<>();
 
         for (ServiceReference<Migratable> serviceReference : getMigratableServiceReferences()) {
-
             Migratable migratable = getBundleContext().getService(serviceReference);
+
+            LOGGER.debug("Exporting {}", migratable.getDescription());
+
+            Stopwatch stopwatch = Stopwatch.createStarted();
```

Comment:
Wrap the creation of this stopwatch in a test for `LOGGER.isDebugEnabled()` and likewise its use on line 147.